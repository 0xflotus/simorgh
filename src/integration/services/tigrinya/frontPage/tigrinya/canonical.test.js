/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service tigrinya
 * @pathname /tigrinya
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical tigrinya frontPage', runCanonicalTests);
