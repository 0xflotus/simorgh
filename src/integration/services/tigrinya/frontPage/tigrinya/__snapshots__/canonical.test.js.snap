// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical tigrinya frontPage A11y I can see the skip to content link 1`] = `"ናብቲ ትሕዝቶ ቀጽሉ"`;

exports[`Canonical tigrinya frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=91&amp;p=tigrinya.page&amp;x1=[urn:bbc:cps:506f0b67-46d6-38d9-e050-17ac80455ed3]&amp;x2=[responsive]&amp;x3=[news-tigrinya]&amp;x4=[ti]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[ዜና+-+BBC+News+ትግርኛ]&amp;x11=[1970-01-18T07:30:12.722Z]&amp;x12=[1970-01-19T02:08:40.352Z]\\"/>"`;

exports[`Canonical tigrinya frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/tigrinya"`;

exports[`Canonical tigrinya frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ቢቢሲ፡ ንትሕዝቶ ካልኦት መርበባት ሓበሬታ ሓላፍነት ኣይወስድን። ብዛዕባ ምስ ናይ ደገ መርበባት እንገብሮ መላግቦታት ዘለና ኣረኣእያ ንምርዳእ ኣንብቡ።"`;

exports[`Canonical tigrinya frontPage Header I can see the branding 1`] = `"BBC News, ትግርኛ"`;

exports[`Canonical tigrinya frontPage I can see Index Alsos 1`] = `"ተዛማዲ ትሕዝቶ"`;

exports[`Canonical tigrinya frontPage I can see Index Alsos 2`] = `"ኣዴፓ፡ ህወሓት ኣብታ ሃገር ንዘለዉ ጸገማት ተሓታቲ'ዩ"`;

exports[`Canonical tigrinya frontPage I can see sections 1`] = `"ዜናታት"`;

exports[`Canonical tigrinya frontPage I can see sections 2`] = `"ከይሓልፈኩም"`;

exports[`Canonical tigrinya frontPage I can see sections 3`] = `"ኣዘራረብቲ ዛዕባታት"`;

exports[`Canonical tigrinya frontPage I can see sections 4`] = `"መዛነዪ"`;

exports[`Canonical tigrinya frontPage I can see sections 5`] = `"ጠመተ"`;

exports[`Canonical tigrinya frontPage I can see sections 6`] = `"ዋንጫ ዓለም 2018"`;

exports[`Canonical tigrinya frontPage I can see the headline 1`] = `"BBC News, ትግርኛ - መእተዊ ገጽ"`;

exports[`Canonical tigrinya frontPage SEO Canonical link 1`] = `"http://localhost:7080/tigrinya"`;

exports[`Canonical tigrinya frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical tigrinya frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical tigrinya frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical tigrinya frontPage SEO Heading level 1 1`] = `"BBC News, ትግርኛ - መእተዊ ገጽ"`;

exports[`Canonical tigrinya frontPage SEO Lang attribute 1`] = `"ti"`;

exports[`Canonical tigrinya frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/tigrinya",
    "@type": "WebPage",
    "name": "ዜና",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
      "width": 1024,
    },
    "name": "BBC News ትግርኛ",
    "publishingPrinciples": "https://www.bbc.com/tigrinya/institutional-49283259",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png",
  "url": "http://localhost:7080/tigrinya",
}
`;

exports[`Canonical tigrinya frontPage SEO OG description 1`] = `"“ብድምጽን ስእልን ጽሑፍን ንዝቐርብ: ዕለታውን ሰበር ዜናታትን ዛንታታት ቢቢሲን ንምክትታል፡ ናብ ድሕረ ገጻት መርበብ ሓበሬታና ብጽሑ። ቢቢሲ፡ ርጡብን እሙንን ዘቤታውን ዓለማውን ዜናታትን ኣመለኻኸታን የቕርብ። ከምኡ እውን ዘዛኒ መደባት፡ ንቢዝነስ፡ ሳይንስ፡ ቴክኖሎጂን ጥዕናን ዝምልከት ትሕዝቶታት የቕርብ እዩ’ሞ ንኽትከታትሉና ናብ መርበብ ሓበሬታ ቢቢሲ ንዕድመኩም"`;

exports[`Canonical tigrinya frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png"`;

exports[`Canonical tigrinya frontPage SEO OG image alt 1`] = `"BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage SEO OG locale 1`] = `"ti-ET"`;

exports[`Canonical tigrinya frontPage SEO OG site name 1`] = `"BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage SEO OG title 1`] = `"ዜና - BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical tigrinya frontPage SEO OG url 1`] = `"http://localhost:7080/tigrinya"`;

exports[`Canonical tigrinya frontPage SEO Page title 1`] = `"ዜና - BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical tigrinya frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical tigrinya frontPage SEO Twitter creator 1`] = `"@bbcnews"`;

exports[`Canonical tigrinya frontPage SEO Twitter description 1`] = `"“ብድምጽን ስእልን ጽሑፍን ንዝቐርብ: ዕለታውን ሰበር ዜናታትን ዛንታታት ቢቢሲን ንምክትታል፡ ናብ ድሕረ ገጻት መርበብ ሓበሬታና ብጽሑ። ቢቢሲ፡ ርጡብን እሙንን ዘቤታውን ዓለማውን ዜናታትን ኣመለኻኸታን የቕርብ። ከምኡ እውን ዘዛኒ መደባት፡ ንቢዝነስ፡ ሳይንስ፡ ቴክኖሎጂን ጥዕናን ዝምልከት ትሕዝቶታት የቕርብ እዩ’ሞ ንኽትከታትሉና ናብ መርበብ ሓበሬታ ቢቢሲ ንዕድመኩም"`;

exports[`Canonical tigrinya frontPage SEO Twitter image alt 1`] = `"BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/tigrinya.png"`;

exports[`Canonical tigrinya frontPage SEO Twitter site 1`] = `"@bbcnews"`;

exports[`Canonical tigrinya frontPage SEO Twitter title 1`] = `"ዜና - BBC News ትግርኛ"`;

exports[`Canonical tigrinya frontPage Story Promo I can see a headline 1`] = `"ርሱን መግለጽታት ህወሓትን ኣዴፓን፡ ናበይ ከምርሕ ይኸውን?"`;

exports[`Canonical tigrinya frontPage Story Promo I can see an image 1`] = `
<img
  alt="ደብረጽዮንን ደመቀን"
  class="StyledImg-sc-7vx2mr-0 hIxkbt"
  sizes="(max-width: 600px) 100vw, (max-width: 1008px) 50vw, 496px"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg"
  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/2FDE/production/_107845221_debretsionanddemeke.jpg 660w"
  width="976"
/>
`;
